src/profile-logic/gecko-profile-versioning.js

```diff
@@ -9,6 +9,9 @@
  * run profiler.firefox.com on non-Nightly versions of Firefox, and we want
  * to be able to load old saved profiles, so this file upgrades old profiles
  * to the current format.
+ *
+ * Please don't forget to update the gecko profile format changelog in
+ * `docs-developer/CHANGELOG-formats.md`.
```

Comment:
I think it would be better to have this comment very close to the lines you will touch when you add a new upgrader. So that would be right after the last upgrader before the closing brace. "If you add a new upgrader here, please document the change in `docs-developer/CHANGELOG-formats.md`."

Reply:
That's a good idea. Added it to there as well.